src/keyutil/keyutil.c

```diff
@@ -306,8 +318,8 @@ static int verb_public(int argc, char *argv[], void *userdata) {
 
 static int verb_pkcs7(int argc, char *argv[], void *userdata) {
         _cleanup_(X509_freep) X509 *certificate = NULL;
-        _cleanup_free_ char *pkcs1 = NULL;
-        size_t pkcs1_len = 0;
+        _cleanup_free_ char *pkcs1 = NULL, *content = NULL;
+        size_t pkcs1_len = 0, content_len = 0;
```

Comment:
move content/conten_len to inermost scope?

Reply:
updated, thanks